exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -1377,7 +1386,7 @@ public void serialize(ColumnMetadata_v2 value, JsonGenerator jgen, SerializerPro
    *
    * Difference between v3 and v2 : min/max, type_length, precision, scale, repetitionLevel, definitionLevel
    */
-  @JsonTypeName("v3") public static class ParquetTableMetadata_v3 extends ParquetTableMetadataBase {
+  @JsonTypeName("v3_1") public static class ParquetTableMetadata_v3 extends ParquetTableMetadataBase {
```

Comment:
Can be replaced with `Version` enum (ex: `Version.V1.getName()`), the same in `ParquetTableMetadataBase` class?